package.json

```diff
@@ -29,6 +29,8 @@
     "@types/react-dom": "16.9.8",
     "@types/socket.io-client": "1.4.34",
     "browser-nativefs": "0.11.0",
+    "clsx": "1.1.1",
+    "browser-nativefs": "0.11.0",
```

Comment:
```suggestion
```

Reply:
duplicate

Reply:
Oh dayum, sorry. Tried github online interface for first time for that :\